robot/client/README.md

```diff
@@ -0,0 +1,57 @@
+# Go client
+The Go client for the Viam RDK can function as an SDK to connect to a robot.
+
+# Install
+
+To install the Go SDK, run
+
+	go get go.viam.com/rdk/robot/client
+
+# Basic Usage
+
+To connect to a robot as a client, you should instantiate a client.
+
+	package main
```

Comment:
i dont know where/how this will be render, but typically wrapping code in ``` makes it stand out and gives syntax highlighting

Reply:
done